theano/tests/test_tutorial.py

```diff
@@ -139,16 +139,13 @@ def make_node(x, y):
 
         from theano import gof
         class BinaryDoubleOp(gof.Op):
+        
+            __props__ = ("name", "fn")
+            
```

Comment:
These tests are supposed to be in sync with the tutorials (check the comment at the beginning of the class).
So if you modify them here, you should also update the tutorials in the documentation at the same time.